cmd/smith/main.go

```diff
@@ -58,6 +58,9 @@ func runWithContext(ctx context.Context) error {
 	if *scUrl != "" {
 		scConfig := *config // shallow copy
 		scConfig.Host = *scUrl
+		scConfig.TLSClientConfig = rest.TLSClientConfig{
+			Insecure: true,
+		}
```

Comment:
I think I saw an issue about TLS and certificates with IP instead of a hostname in custom api servers somewhere....